### PR TITLE
feat: integrate formbricks in help feedback box

### PR DESCRIPTION
**Pull request by ShubhamPalriwala**
## What does this PR do?
The Help Feedback Box currently sends the data to our email service. This PR sends it to a pre-created Formbricks Survey for easier visibility and availability in the Formbricks App.

image

We use the `@formbricks/api` package for minimal dependencies and create a response in the app.

## Requirement/Documentation
We introduce a new environment variable that needs to be set:
`NEXT_PUBLIC_FORMBRICKS` in the form of `[apiHost]+[environmentId]+[surveyId]`

For testing, use:
```
https://app.formbricks.com+cloob69in56synz0fg6l5vqmx+cloob6fn356u0nz0fl0mm1wa5
```

Before merging to main, 
- [ ] Create a Survey on Formbricks
- [ ] Create 2 questions (1 Open Text with Question ID as: `formbricks-share-comments-question` and 1 Rating question with Range of 4  and ID as `formbricks-rating-question` )

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

- Set the env variable as suggested above
- Fill the Feedback Box inside the Help Button
- View in the Formbricks App and the response should be reflected

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -115,6 +115,10 @@ SENDGRID_API_KEY=
 SENDGRID_EMAIL=
 NEXT_PUBLIC_SENDGRID_SENDER_NAME=
 
+# Formbricks Experience Management Integration
+# [apiHost]+[environmentId]+[surveyId]
+NEXT_PUBLIC_FORMBRICKS=
```
Comment (mattinannt) [inline .env.example]: I'm not sure if we should mix all of them in one environment variable. If we do we need more information about the part or rename `surveyId` to `feedbackSurveyId` to make this clear

File: packages/lib/formbricks.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+if (!process.env.NEXT_PUBLIC_FORMBRICKS) throw new Error("Missing NEXT_PUBLIC_FORMBRICKS env variable");
+const [apiHost, environmentId, surveyId] = process.env.NEXT_PUBLIC_FORMBRICKS.split("+");
+const api = new FormbricksAPI({
+  apiHost,
+  environmentId,
+});
+
+export const submitSurveyResponse = async (rating: string, comment: string) => {
+  console.log(Rating[rating]);
```
Comment (mattinannt) [inline packages/lib/formbricks.ts]: please remove the console.log

File: packages/lib/formbricks.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+if (!process.env.NEXT_PUBLIC_FORMBRICKS) throw new Error("Missing NEXT_PUBLIC_FORMBRICKS env variable");
+const [apiHost, environmentId, surveyId] = process.env.NEXT_PUBLIC_FORMBRICKS.split("+");
+const api = new FormbricksAPI({
+  apiHost,
+  environmentId,
+});
+
+export const submitSurveyResponse = async (rating: string, comment: string) => {
```
Comment (mattinannt) [inline packages/lib/formbricks.ts]: please rename to `submitFeedback` or similar to make clear that this is to submit the feedback from the feedback box

Comment (ShubhamPalriwala) [reply packages/lib/formbricks.ts]: renamed it to `submitFeedbackFormbricks` to maintain their pattern of `sendFeedbackEmail` for email

File: .env.example (hunk 1 above)
Comment (ShubhamPalriwala) [reply .env.example]: Done to:
```
# [apiHost]+[environmentId]
FORMBRICKS=
# Formbricks Feedback Survey ID
FORMBRICKS_FEEDBACK_SURVEY_ID=

```

**Pull request closed without merging**